### PR TITLE
Able to create pins without lat/long provided

### LINKED ISSUE #233: Able to create a pin without lat/long

### PR DESCRIPTION
**Pull request by supasate**
Fix #233

### CODE REVIEW

File: src/services/pin/pin-model.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ const mongoose = require('mongoose');
 
 const PENDING = require('../../constants/pin-states').PENDING;
 
+// default to Thailand Democracy Monument
+const DEFAULT_LAT_LONG = [100.5018549, 13.756727];
```
Comment (parnurzeal) [inline src/services/pin/pin-model.js]: Put it to constant folder?

**Comment by parnurzeal** [comment]
Isn't we able to create pin w/o location already? (because we have default lat-long in model.)

**Comment by supasate** [comment]
It dies in `swapLatLongHelper`.

File: src/services/pin/pin-model.js (hunk 1 above)
Comment (supasate) [reply src/services/pin/pin-model.js]: What about `src/constants/default.js`?

Comment (parnurzeal) [reply src/services/pin/pin-model.js]: Yes.

**Comment by parnurzeal** [comment]
I see. :'(

**Comment by supasate** [comment]
Done

**Pull request merged**